### PR TITLE
Add Pocket Card Jockey

### PR DESCRIPTION
**Pull request by MelonSpeedruns**
Adds Pocket Card Jockey.

*Automatically generated by PR builder*

### CODE REVIEW

File: games/pocket-card-jockey/game.dat [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+[[releases]]
+title = "0004000000194A00"
+region = "kor"
+release_date = "2016-11-09"
+
+[[releases]]
+title = "00040000000EB000"
+region = "jpn"
+release_date = "2016-07-31"
```
Comment (Leo626) [inline games/pocket-card-jockey/game.dat]: `2013-07-31`

File: games/pocket-card-jockey/game.dat [hunk 2]
```diff
@@ -0,0 +1,25 @@
+title = "Pocket Card Jockey"
+description = "Pocket Card Jockey (SolitiBa in Japan) is a horse racing game by Game Freak. The game combines horse racing and simplified solitaire."
```
Comment (Hexagon12) [inline games/pocket-card-jockey/game.dat]: _(Solitaiba in Japan)_

Comment (Leo626) [reply games/pocket-card-jockey/game.dat]: I've seen [GameFAQs](https://gamefaqs.gamespot.com/3ds/724556-pocket-card-jockey) call it SolitiBa. Other places I've seen it:

http://nintendo.wikia.com/wiki/Pocket_Card_Jockey
http://pocketcardjockey.wikia.com/wiki/Pocket_Card_Jockey_Wikia
http://www.mobygames.com/game/pocket-card-jockey

**Comment by Leo626** [comment]
@MelonSpeedruns Fix the jpn release date and then I'll merge it.

**Pull request merged**